themes/app/templates/Includes/Header.ss

```diff
@@ -30,8 +30,8 @@
                 <% end_loop %>
             </div>
         </div>
-        <div slot="button" class="main-nav__donate">
-            <button class="btn btn-outline-secondary" type="button">DONATE NOW</button>
+        <div slot="button" class="main-nav__button">
+            <a class="btn btn-outline-secondary" href="$SiteConfig.HeaderButtonLink.Link" role="button">$SiteConfig.HeaderButtonText</a>
```

Comment:
Nice! I like this line, might have to steal it for myself 😉

Reply:
😘